shuup/front/templates/shuup/front/macros/order.jinja

```diff
@@ -146,6 +150,27 @@
     </div>
 {% endmacro %}
 
+{% macro render_line_properties(properties) %}
```

Comment:
You now have this macro defined twice it seems. Not sure if bad idea since I think basket and order info rendering could be separated. Changing other shouldn't necessary change the other.